docs/index.rst

```diff
@@ -1843,7 +1843,7 @@ Later, you can access the story's state when creating other fixtures:
         use Zenstruck\Foundry\Story;
 
         /**
-         * @method php(): Category
+         * @method php() Category
```

Comment:
```suggestion
         * @method Category php()
```

?

Reply:
[It look](https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/method.html) looks like the method name are coming after the return type but its not working. The proposed change is tested successful